src/main/java/com/bina/varsim/types/Parent_record.java

```diff
@@ -1,6 +1,11 @@
 package com.bina.varsim.types;
 
 
+import com.bina.varsim.types.stats.Type_record;
```

Comment:
Yeah, I agree, but there are so many places to change this and it may break the templates in the javascript